src/pyFAI/integrator/common.py

```diff
@@ -137,6 +137,7 @@ def __init__(self, dist=1, poni1=0, poni2=0,
 
         self._lock = threading.Semaphore()
         self.engines = {}  # key: name of the engine,
+        self._cache_parameters = {}
```

Comment:
is it ok to this at this level?
it helps me with caching the grazing angles

Reply:
with an __init__ method ?

Reply:
Yes, just call the parent's constructor and it will work.